src/snowflake/snowpark/udtf.py

```diff
@@ -421,6 +421,98 @@ class UDTFRegistration:
             -----------------------------
             <BLANKLINE>
 
+    [Preview Feature] Syntax for declaring vectorized UDTF process method is similar to above. Defining
+    ``__init__`` and ``end_partition`` methods are optional. ``process`` method only accepts one argument
+    which is the pandas dataframe object. Both ``__init__`` and ``end_partition`` do not take any additional
+    arguments.
+
```

Comment:
@sfc-gh-sabdelrahman can you review the doc strings

Reply:
Can we also mention the added restriction that it has to be 1-1 mapping functions? 
That is the # of output rows for each call has to match the # of input rows (similar to scalar vUDFs)

Reply:
This restriction is for `process` method right. Should I also add any comment about the output size of `end_partition`.

Reply:
No it's only for `process`.  `end_partition` can return any number or rows.

Reply:
@sfc-gh-mabrennan your inputs here would be useful as well